doc/masternode_conf.md

```diff
@@ -1,30 +1,29 @@
-Multi masternode config
+Masternode config
 =======================
 
-The multi masternode config allows to control multiple masternodes from a single wallet. The wallet needs to have a valid collaral output of 1000 coins for each masternode. To use this, place a file named `masternode.conf` in the data directory of your install:
+Dash Core allows to control multiple remote masternodes from a single wallet. Such wallet needs to have a valid collaral output of 1000 coins for each masternode and uses a configuration file named `masternode.conf` which can be found in the following data directory (depending of your operating system):
```

Comment:
s/collaral/collateral
/

Better English:
s/allows to control/allows controlling/
s/Such wallet/The wallet/
s/depending of/depending on/